### PR TITLE
Improve delegate view

### LINKED ISSUE #248: Set current election as default value selected
Image

### LINKED ISSUE #251: Empty states
As a user, when I choose a delegate who is not in the election, the program does not show me any information

We could place a empty stated where it indicates that the selected delegate was not elected in that specific election.

https://spending.eden.eoscommunity.org/delegate

you can find info in this link

https://www.toptal.com/designers/ux/empty-state-ux-design#:~:text=What%20are%20empty%20states%3F,common%20types%20of%20empty%20states.

Image

### LINKED ISSUE #260: Improve Delegate View
In the delegate section At this moment when the information is displayed there is a box and a line as you can see in the image

https://spending.eden.eoscommunity.org/delegate

Image

we want to remove that line when displaying the delegates, but we want to keep it when displaying the charts
as the image shows

Image

### PR DESCRIPTION
**Pull request by JAMares**
### What does this PR do?

- Resolve #248
- Resolve #260 
- Resolve #251 

### Steps to test

1. Run project
2. Enter to Delegate view

#### CheckList

- [X] Follow proper Markdown format
- [X] The content is adequate
- [X] The content is available in both english and spanish
- [X] I Ran a spell check

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -66,12 +66,12 @@ services:
...
       HASURA_GRAPHQL_ENABLE_CONSOLE: 'true'
       # Local Development Configurations It is recommended to disable this in production
-      # HASURA_GRAPHQL_DEV_MODE: 'true'
+      HASURA_GRAPHQL_DEV_MODE: 'true'
       HASURA_GRAPHQL_ENABLED_LOG_TYPES: startup, http-log, webhook-log, websocket-log, query-log
       HASURA_GRAPHQL_ENABLE_LOGGING: 'true'
       # disabled security for local development
-      HASURA_GRAPHQL_ADMIN_SECRET: '${HASURA_GRAPHQL_ADMIN_SECRET}'
-      HASURA_GRAPHQL_UNAUTHORIZED_ROLE: '${HASURA_GRAPHQL_UNAUTHORIZED_ROLE}'
+      # HASURA_GRAPHQL_ADMIN_SECRET: '${HASURA_GRAPHQL_ADMIN_SECRET}'
```
Comment (AngeloCG97) [inline docker-compose.yaml:73]: We should'n comment this vars

File: hapi/src/services/delegate-frontend.service.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  const dataDelegate = newDataFormatDelegatesByElectionDelegate({
+    delegateList: delegates,
+    incomeList: incomeByElection
+  })
+  dataDelegate.push(dateElection)
+  dataDelegate.push(levelElection)
+  const dataFound = {
+    data: dataDelegate
+  }
+  return dataFound
```
Comment (AngeloCG97) [inline hapi/src/services/delegate-frontend.service.js:41]: Please add space before return

File: hapi/src/services/delegate-frontend.service.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+const newDataFormatDelegatesByElectionDelegate = ({
+  delegateList,
+  incomeList
+}) => {
+  if (delegateList !== undefined && incomeList !== undefined) {
+    return delegateList.map(delegate => {
+      const posDelegate = incomeList.find(
+        income => income.recipient === delegate.delegate_payer
+      )
+      if (posDelegate) {
```
Comment (AngeloCG97) [inline hapi/src/services/delegate-frontend.service.js:12]: Please add space before and after if and return

**Review by AngeloCG97 — CHANGES_REQUESTED** [review]

File: webapp/src/language/en.json [hunk 4]
```diff
@@ -30,7 +30,7 @@
     "changelog": "Changelog",
     "github": "Github Repository",
     "telegram": "Telegram Channel",
-    "invalidEdenMember": "Invalid user, this aren't Eden member"
+    "invalidEdenMember": "Invalid user, this is't Eden member"
```
Comment (xavier506) [inline webapp/src/language/en.json:33]: Invalid user, this account isn't an Eden member

**Review by AngeloCG97 — APPROVED** [review]
LGTM

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**